app/views/eventlist.scala.html

```diff
@@ -0,0 +1,36 @@
+@(events: List[Event])
+@base(title = "Live Events") {
+
+    <table class="dashboardpanel">
```

Comment:
absolutely... I've not used a flexbox before though. can we accept this then PR an update next?